src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -256,20 +239,12 @@ NavHeader.propTypes = {
 };
 
 const styles = StyleSheet.create({
-  container: (animatedHeight) => ({
-    width: '100%',
-    height: animatedHeight,
-    backgroundColor: COLORS.gray110,
-    position: 'absolute',
-    zIndex: 80,
-    overflow: 'hidden'
-  }),
-  navContainer: (isHeaderHide, headerHeight) => ({
+  navContainer: (isHeaderHide, additionalHeight) => ({
     flexDirection: 'row',
     justifyContent: 'space-between',
-    alignItems: 'center',
+    alignItems: 'flex-start',
     zIndex: 10,
-    height: headerHeight,
+    height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER + additionalHeight,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in the `navContainer` style seem to be related to the calculation of the height based on the `additionalHeight`. As mentioned before, make sure this logic is well-tested and documented.



<!-- This is an auto-generated comment by OSS CodeRabbit -->